src/NServiceBus.Testing.Tests/SagaTests.cs

```diff
@@ -22,6 +22,22 @@ public void MySaga()
         }
 
         [Test]
+        public void MySagaWithConstructorDependency()
+        {
+            var instance = new MySagaWithConstructorDependency(null);
```

Comment:
Thanks! After checking with my employer, I've e-signed the agreement. I'll
clean up this PR later.

- Jason

On Wed, Dec 9, 2015 at 8:19 AM, Tim Bussmann <notifications@github.com>
wrote:

> In src/NServiceBus.Testing.Tests/SagaTests.cs
> <https://github.com/Particular/NServiceBus.Testing/pull/28#discussion_r47095006>
> :
>
> > @@ -22,6 +22,22 @@ public void MySaga()
> >          }
> >
> >          [Test]
> > +        public void MySagaWithConstructorDependency()
> > +        {
> > +            var instance = new MySagaWithConstructorDependency(null);
>
> I think it would be nicer if the passed dependency wouldn't be null. You
> could easily change the dependency from an interface to a class and new
> that up.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Particular/NServiceBus.Testing/pull/28/files#r47095006>
> .
>